.env.development

```diff
@@ -3,6 +3,7 @@ VITE_HOTJAR_ID=3307684
 VITE_HOTJAR_SNIPPET_VERSION=6
 VITE_LIFI_API_URL=https://develop.li.quest/v1
 VITE_WIDGET_INTEGRATOR=dev.jumper.exchange
+VITE_WIDGET_INTEGRATOR_REFUEL=dev.jumper.exchange.gas
```

Comment:
Also update the env declaration file accordingly.

Reply:
Done thanks